browser-test/src/admin/admin_application_statuses.test.ts

```diff
@@ -359,6 +361,171 @@ test.describe('view program statuses', () => {
+        const {page} = ctx
+        await disableFeatureFlag(
+          page,
+          'primary_applicant_info_questions_enabled',
+        )
+      })
+      test('email is displayed and sent for guest user that has answered the PAI email question', async () => {
+        const {page, applicantQuestions, adminPrograms} = ctx
+        const guestEmail = 'guestemail@example.com'
+
+        await applicantQuestions.applyProgram(programWithStatusesName)
+        await applicantQuestions.answerEmailQuestion(guestEmail)
+        await applicantQuestions.clickNext()
+        await applicantQuestions.submitFromReviewPage()
+        await logout(page)
+
+        await loginAsProgramAdmin(page)
+        await adminPrograms.viewApplications(programWithStatusesName)
+        await adminPrograms.viewApplicationForApplicant('Guest')
+
+        const emailsBefore = supportsEmailInspection()
+          ? await extractEmailsForRecipient(page, guestEmail)
+          : []
+
+        const modal =
+          await adminPrograms.setStatusOptionAndAwaitModal(emailStatusName)
+
+        const notifyCheckbox = await modal.$('input[type=checkbox]')
+        if (!notifyCheckbox) {
+          throw new Error('Expected a checkbox input')
+        }
+        expect(await notifyCheckbox.isChecked()).toBe(true)
+
+        expect(await modal.innerText()).toContain(
+          ' of this change at ' + guestEmail,
+        )
+        await adminPrograms.confirmStatusUpdateModal(modal)
+        expect(await adminPrograms.getStatusOption()).toBe(emailStatusName)
+        await adminPrograms.expectUpdateStatusToast()
+
+        if (supportsEmailInspection()) {
+          await adminPrograms.expectEmailSent(
+            emailsBefore.length,
+            guestEmail,
+            emailBody,
+            programWithStatusesName,
+          )
+        }
+      })
+      test('both email addresses are displayed and two emails are sent for a logged in user that has answered the PAI email question with a different email', async () => {
+        const {page, applicantQuestions, adminPrograms} = ctx
+        const otherTestUserEmail = 'other@example.com'
+
+        await loginAsTestUser(page)
+        await applicantQuestions.applyProgram(programWithStatusesName)
+        await applicantQuestions.answerEmailQuestion(otherTestUserEmail)
+        await applicantQuestions.clickNext()
+        await applicantQuestions.submitFromReviewPage()
+        await logout(page)
+
+        await loginAsProgramAdmin(page)
+        await adminPrograms.viewApplications(programWithStatusesName)
+        await adminPrograms.viewApplicationForApplicant(testUserDisplayName())
+
+        const acccountEmailsBefore = supportsEmailInspection()
+          ? await extractEmailsForRecipient(page, testUserDisplayName())
+          : []
+        const applicantEmailsBefore = supportsEmailInspection()
+          ? await extractEmailsForRecipient(page, otherTestUserEmail)
+          : []
```

Comment:
This looks like it could go inside the `if (supportsEmailInspection)` block starting on line 476.

Reply:
This actually has to be on it's own to capture the number of emails before we do another send.

Reply:
Putting that in a `test.step('Gather emails before doing the thing you are going')` would be a boon.

Reply:
How you can return the const variables from a step https://timdeschryver.dev/blog/keep-your-playwright-tests-structured-with-steps#steps

Reply:
Done.